src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -119,6 +119,47 @@
         az webapp config appsettings set -g MyResourceGroup -n MyUniqueApp --settings WEBSITE_NODE_DEFAULT_VERSION=6.9.1
 """
 
+helps['webapp config azure-storage-accounts'] = """
```

Comment:
what about function app?

Reply:
I would suggest `az webapp config storage-acount` as `storage account` pretty much means `azure storage account` . Generally, `azure` is a redundant term which I suggest remove from command/argument namings to get things simpler. Also avoid plurals in the command naming.

Reply:
the functionapp runtime doesn't use this, so we dont need to add it

Reply:
@patricklee2 , i am guessing you mean the feature is not supported on function app as yet. If so yes, don't need to add support here. Please do update the command name as per Yugang's comments.